packaging/macos/tests/run_tests.sh

```
@@ -0,0 +1,158 @@
+  if [[ "${VERBOSE}" == "true" ]]; then
+    printf 'CASE  %s :: %s\n' "${CUR_TEST_FILE}" "${name}"
+  fi
+  local before_fail=${TEST_FAIL_COUNT}
+  if "$@"; then : ; fi
+  if [[ ${TEST_FAIL_COUNT} -eq ${before_fail} ]]; then
+    TEST_OK_COUNT=$((TEST_OK_COUNT + 1))
+    if [[ "${VERBOSE}" == "true" ]]; then
+      printf '  PASS\n'
+    fi
+  fi
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)